### PR TITLE
genetics power that lets you fire a bullet by throwing it

### PR DESCRIPTION
**Pull request by Simplehorror**
# Document the changes in your pull request

because it's cool
# Spriting

# Wiki Documentation

name: trigger finger
desc: This mutation lets you fire bullets simply by flicking them.
instability: 20
positive
# Changelog

:cl:  
rscadd: genetics power that lets you fire a bullet by throwing it
/:cl:

### CODE REVIEW

**Review by ToasterBiome — COMMENTED** [review]
make it break your arm unless you reinforce it

**Comment by Simplehorror** [comment]
reinforcements are being removed soon

**Comment by Bibby0110** [comment]
> reinforcements are being removed soon

no they aren’t

**Comment by Simplehorror** [comment]
#16773

**Comment by UltraBigRat** [comment]
> make it break your arm unless you reinforce it

Would be a pretty sucky way of balancing it, if you want to balance it itd be better to just make it deal brute to the arm depending on the caliber (so things like chucking 38 does barely any harm while) while shotgun shells will deal high damage to your arm. I feel with that itd be pretty good for antags, theres alot of validhunting potential there though. With shit like geneticists running around  with a box of buckshot and shotgun hands

**Comment by ReddicusDragon** [comment]
holy shit this is actually funny

File: code/datums/mutations/body.dm [hunk 1]
```diff
@@ -513,14 +513,9 @@
 	var/obj/item/currently_held = owner.get_active_held_item()
 	if (istype(currently_held, /obj/item/ammo_casing))
 		var/obj/item/ammo_casing/casing = currently_held
-
-		if (casing.BB)
```
Comment (ReddicusDragon) [inline code/datums/mutations/body.dm]: can you please show me where BB is defined I am having a dumb moment

Comment (ReddicusDragon) [reply code/datums/mutations/body.dm]: found it

**Review by ReddicusDragon — APPROVED** [review]
Code looks good

**Comment by MajManatee** [comment]
cant wait to make pulse slugs and shoot them at antags with nothing but my hands

**Comment by ToasterBiome** [comment]
> > make it break your arm unless you reinforce it
> 
> Would be a pretty sucky way of balancing it, if you want to balance it itd be better to just make it deal brute to the arm depending on the caliber (so things like chucking 38 does barely any harm while) while shotgun shells will deal high damage to your arm. I feel with that itd be pretty good for antags, theres alot of validhunting potential there though. With shit like geneticists running around with a box of buckshot and shotgun hands

who cares about balance (not me)

File: code/datums/mutations/body.dm (hunk 1 above)
Comment (Simplehorror) [reply code/datums/mutations/body.dm]: jeez how did you get to this

**Comment by Simplehorror** [comment]
it should hurt your index finger

**Comment by Sniblet** [comment]
I put the meteorshot in my mouth and bite down

**Comment by kooarbiter** [comment]
literally catch these hands

**Comment by 00ze-cyclone** [comment]
Absolutely no way lizbay will abuse the fuck out of this by printing mosin bullets with the autolathe they build at the start of every round yes yes.
Very useful mutation for me the non antag geneticists/MD/scientist/assistant because yeah I definitely need an undroppable gun and it’ll certainly not be used to validhunt.
Let’s give everyone a makeshift gauss rifle except concealable, or a makeshift shotgun with no jam chances, what could go wrong, nothing of course.
Excellent PR and will greatly improve the game.

**Review by ReddicusDragon — CHANGES_REQUESTED** [review]
Upon re-examining this PR, I suggest making a chance to fail and or a chance to just completely break your arm.

File: code/datums/mutations/body.dm [hunk 2]
```diff
@@ -496,3 +496,28 @@
 		return
 	owner.physiology.stamina_mod /= 0.7
 	owner.physiology.stun_mod /= 0.85
+
+/datum/mutation/human/trigger_finger
+	name = "Trigger Finger"
+	desc = "This mutation lets you fire bullets simply by flicking them."
+	quality = POSITIVE
+	difficulty = 16
+	instability = 20
```
Comment (MajManatee) [inline code/datums/mutations/body.dm:505]: Should probably be increased, at least in its current state.

**Comment by chemistrymain2** [comment]
> Absolutely no way lizbay will abuse the fuck out of this by printing mosin bullets with the autolathe they build at the start of every round yes yes. Very useful mutation for me the non antag geneticists/MD/scientist/assistant because yeah I definitely need an undroppable gun and it’ll certainly not be used to validhunt. Let’s give everyone a makeshift gauss rifle except concealable, or a makeshift shotgun with no jam chances, what could go wrong, nothing of course. Excellent PR and will greatly improve the game.

honestly, while all of these are genuine concerns dude i doubt this is a very effective powergamey power due to how slow it is to pull a bullet out and then throw it, and how rapidly full your inventory would become

i'd personally say give it a downside like damaging your arm or costing stamina to do so per bullet, but if you're looking for powergame, 'guy throwing one bullet at a time at me' isn't significantly that powerful when literally anyone can get their hands on roughly equivalent bullshit with minimal effort themselves if they so choose to

obviously, it becomes more problematic with things like shotgun shells or mosin rounds, as you've pointed out, as that does hit very hard, so another alternative is to simply have increasing penalties the more powerful the bullet is-a single 10mm being less of an issue to propel than, say, a pulse round or mosin shot, which are very strong

the core idea is a cool concept, it just needs balancing

**Comment by chemistrymain2** [comment]
> Upon re-examining this PR, I suggest making a chance to fail and or a chance to just completely break your arm.

personally i'd say either there be a hard cap on the strength of the allowed projectile, a maximum damage for the allowed projectile, or just outright costing stamina and brute to use said projectiles in combination with the other two (or you could indeed go the whole mile and have sufficiently powerful projectiles outright break your arm)

an RNG chance is the least pleasant balancing method there is and ideally RNG goes away from combat, so a random chance to just break your arm makes it annoying for both user and victim,whereas if someone _knows_ they can't get away with using a buckshot shell without breaking their arm and knocking themselves down, that absolutely balances it out

**Review by Moltijoe — CHANGES_REQUESTED** [review]
Needs a drawback, 20 instability to basically get all ballistic guns (assuming you can get the ammo) is too strong since autolathes exist

Self Damage if not powered
limit on certain bullets it doesn't work for
Thank kinda stuff

**Comment by Simplehorror** [comment]
what kinda bullets

**Comment by Moltijoe** [comment]
> what kinda bullets

figure it out

**Comment by isotope436** [comment]
https://www.youtube.com/watch?v=1K17tnZ9S6Y

**Comment by MajManatee** [comment]
Increase the instability, and maybe give it a cooldown. This is basically the balistic version of lizard firebreath

**Comment by Simplehorror** [comment]
well humans dont have a race specific thing anymore, it is literally more advantageous to play lizard right now

**Comment by UltraBigRat** [comment]
> well humans dont have a race specific thing anymore, it is literally more advantageous to play lizard right now

They do, its called asimov ai

**Comment by Bibby0110** [comment]
stale

**Pull request closed without merging**